ui/lib/core/addon/templates/components/replication-dashboard.hbs

```diff
@@ -9,16 +9,41 @@
         data-test-isReindexing />
     </div>
   {{/if}}
-  <div
-    class="selectable-card-container {{if isSecondary 'secondary' 'primary'}}"
-    data-test-selectable-card-container={{if isSecondary 'secondary' 'primary'}}>
-    {{yield (hash
-      card=(component componentToRender replicationDetails=replicationDetails)
-    )}}
-    {{#if (not isSecondary)}}
-      {{yield (hash secondaryCard=(component 'known-secondaries-card'))}}
-    {{/if}}
-  </div>
+  {{#if isSummaryDashboard}}
+    <div class="summary-state">
+      <h6 class="title is-5 {{if (get (cluster-states summaryState) "isOk") "" "has-text-danger"}}" data-test-error>state</h6>
```

Comment:
Very small nit, but this ternary could be rewritten as
`{{unless (get (cluster-states summaryState) "isOk") "has-text-danger"}}`